apps/meteor/tests/e2e/page-objects/omnichannel-business-hours.ts

```diff
@@ -0,0 +1,67 @@
+import type { Locator } from '@playwright/test';
+
+import { OmnichannelAdministration } from './omnichannel-administration';
+
+export class OmnichannelBusinessHours extends OmnichannelAdministration {
+  get btnCreateBusinessHour(): Locator {
+		return this.page.locator('header').locator('role=button[name="New"]');
+	}
+
+  get btnSave(): Locator {
+		return this.page.locator('role=button[name="Save"]');
```

Comment:
I dont see a lot of value creating functions just to wrap a button role. it may be a bit more contextualized  (page> main> ...

Reply:
Yes, I know that for every selector there is a function, if not more than one. But I was wondering if we shouldn't avoid it